spec/models/collection_spec.rb

```diff
@@ -82,6 +82,22 @@
       expect(collection.errors.full_messages).to \
         include /Sorry, a collection can only have 10 tags./
     end
+
+    it "disallows invalid header image urls" do
+      collection.header_image_url = "https://example.com/image.webp"
+      expect(collection.save).to be_falsey
```

Comment:
Since we're just testing validations, you could use `.valid?`, which is faster than saving the record.